2020-02-Spring/0_arrays_matrix/README.md

```diff
@@ -119,21 +125,43 @@ Go to [Top](#top)
 <!-- Don't remove -->
 <a name="s2"/>
 
-### 2. SOLUTION 2 TODO :bug:
-
-Source: TODO :bug:
-
-#### Naive/Simple Solution
-
-TODO :bug:
-
-#### Optimal Solution
-
-TODO :bug:
-
-#### Testing The Solutions OR Driver For Solution
-
-TODO :bug:
+### 2. SOLUTION 2
+
+Source: https://leetcode.com/problems/maximum-subarray/
+
+We will traverse over the array and keep track of 2 variables: Total maximum sum, and the current sum.
+We will look at each number and add it to the current sum, if the current sum becomes negative, we ignore the numbers that made it negative.
+Resetting the current to 0 is basically saying "Ignore everything we looked at previously because it's not getting us the greatest sum."
+So if we are looking at this array: `[-2,1,-3,4,-1,2,1,-5,4]`
```

Comment:
This renders as one large paragraph; recommend breaking it up into explanation + example.

![large_paragraph](https://user-images.githubusercontent.com/29528527/78851272-3ead7380-79ce-11ea-80cf-ed49daa69653.png)


```suggestion

So if we are looking at this array: `[-2,1,-3,4,-1,2,1,-5,4]`
```

Reply:
You should still do this

Reply:
Fixed in next commit